src/main/resources/db/migration/V11__monitoring_configuration.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE hydro_serving.model_version ADD COLUMN monitoring_configuration JSON NOT NULL DEFAULT '{"batch_size": 100}'::json;
```

Comment:
for backward compat we need to put 10 here
this ensures that old models that are already deployed won’t break (hopefully)

Reply:
Done